selfdrive/car/car_specific.py

```diff
@@ -88,6 +88,23 @@ def update(self, CS: CarStateBase, CS_prev: car.CarState, CC_prev: car.CarContro
       if self.CP.minEnableSpeed > 0 and CS.out.vEgo < 0.001:
         events.add(EventName.manualRestart)
 
+      # Low speed steer alert logic; for cars with steer cut off above 6 m/s
+      # Show the alert earlier when slowing. Must first exceed a greater speed while engaged.
+      if CS.out.vEgo > (self.CP.minSteerSpeed + 3.5):
```

Comment:
I chose this policy as users have reported that continuing to see the alert just after the car has began to steer can be confusing.